pom.xml

```diff
@@ -602,6 +602,7 @@
                                     <dependency>log4j:log4j</dependency>
                                     <!-- AssertJ -->
                                     <dependency>org.assertj:assertj-core</dependency>
+                                    <dependency>org.apache.camel.quarkus:camel-quarkus-beanio</dependency>
```

Comment:
The camel-quarkus-beanio extension seems to have incomplete metadata. At this stage, the plan would be to add it back in a future release once the root cause has been identified.

Reply:
Many thanks to @jamesnetherton who found the root [cause](https://github.com/apache/camel-quarkus/issues/5803). It's probably time to add a comment pointing to this issue + squashing commits.